subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -196,13 +208,16 @@ static uint8_t force_md_cnt_calc(struct lll_conn *lll_conn, uint32_t tx_rate);
 /* CIS Establishment procedure state values */
 #define CIS_REQUEST_AWAIT_HOST 2
 
-static MFIFO_DEFINE(conn_tx, sizeof(struct lll_tx), CONFIG_BT_BUF_ACL_TX_COUNT);
+static MFIFO_DEFINE(conn_tx, sizeof(struct lll_tx),
+		    (CONFIG_BT_BUF_ACL_TX_COUNT + CONN_ENC_REQ_BUFFERS));
 static MFIFO_DEFINE(conn_ack, sizeof(struct lll_tx),
-		    (CONFIG_BT_BUF_ACL_TX_COUNT + CONN_TX_CTRL_BUFFERS));
+		    (CONFIG_BT_BUF_ACL_TX_COUNT + CONN_ENC_REQ_BUFFERS +
+		     CONN_TX_CTRL_BUFFERS));
 
 static struct {
 	void *free;
-	uint8_t pool[CONN_TX_BUF_SIZE * CONFIG_BT_BUF_ACL_TX_COUNT];
+	uint8_t pool[CONN_TX_BUF_SIZE *
+		     (CONFIG_BT_BUF_ACL_TX_COUNT + CONN_ENC_REQ_BUFFERS)];
```

Comment:
Given that `CONFIG_BT_BUF_ACL_TX_COUNT + CONN_ENC_REQ_BUFFERS` is used that many places now, would it make sense to add a `#define` for that value, and use that?

Reply:
done